migrations/sql/V2019.03.25.11.06__add_hsrc_code.sql

```diff
@@ -0,0 +1,1069 @@
+CREATE TABLE IF NOT EXISTS article_act_code
+(
+    article_act_code character varying(5)     PRIMARY KEY                        ,
+    description      character varying(100)                              NOT NULL,
+    effective_date   date                     DEFAULT now()              NOT NULL,
+    expiry_date      date                     DEFAULT '9999-12-31'::date NOT NULL,
+    update_timestamp timestamp with time zone DEFAULT now()              NOT NULL
+);
+
+
+-- Comments
+COMMENT ON TABLE article_act_code IS 'Contains a list of values used to indicate an Act or Regulation and is used to define the article_act_code. I.e. MA - Mines Act, HSRC - Health, Safety and Reclamation Code for Mines in BC.';
+COMMENT ON COLUMN article_act_code.description IS 'This is the legal name of the Act or Regulation.';
+COMMENT ON COLUMN article_act_code.effective_date IS 'Effective Date reflects the date the Act or Regulation is legally enacted.';
+COMMENT ON COLUMN article_act_code.expiry_date IS 'Expiry Date reflects the legal date an Act or Regulation is no longer valid. 9999-12-31 implies the code value is still valid.';
+
+
+INSERT INTO article_act_code
+(
+    article_act_code,
+    description,
+    effective_date,
+    expiry_date,
+    update_timestamp
+)
+VALUES
+(
+    'HSRCM',
+    'Health, Safety and Reclamation Code for Mines in BC (MA)',
+    '1970-01-01',
+    '9999-12-31',
+    '2018-05-15'
+),
+(
+    'MA',
+    'Mines Act (293/1996)',
+    '1970-01-01',
+    '9999-12-31',
+    '2018-05-15'
+);
+
+CREATE TABLE IF NOT EXISTS compliance_article
+(
+    compliance_article_id integer                  PRIMARY KEY                        ,
+    article_act_code      character varying(5)                                NOT NULL,
+    section               character varying(6)                                NOT NULL,
+    sub_section           character varying(4)                                        ,
+    paragraph             character varying(4)                                        ,
+    sub_paragraph         character varying(4)                                        ,
+    description           character varying(100)                              NOT NULL,
+    effective_date        date                     DEFAULT now()              NOT NULL,
+    expiry_date           date                     DEFAULT '9999-12-31'::date NOT NULL
+);
```

Comment:
I forgot to mention that we may want the 4 audit columns on this table (create_user, create_timestamp, update_user, update_timestamp)

Reply:
Added 👍